### ISSUE TITLE
Example fails

### ISSUE THREAD

**Issue by TimMensch** [uhyo/nitrogql#52]
I was trying to track down why I couldn't use `nitrogql generate` and I gave up on figuring out what was wrong with my configuration and cloned the nitrogql Github.

The `example/vite` folder results in this:

```
$ npm i

added 57 packages, and audited 58 packages in 42s

5 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

$ npm run generate

> vite@1.7.0 generate
> nitrogql generate  

TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file
    at fileURLToPath (node:internal/url:1463:11)
    at load (file:///E:/projects/nitrogql-example-vite/node_modules/@nitrogql/esbuild-register/dist/hook.mjs:66:42)
    at nextLoad (node:internal/modules/esm/hooks:866:28)
    at Hooks.load (node:internal/modules/esm/hooks:449:26)
    at handleMessage (node:internal/modules/esm/worker:196:24)
    at Immediate.checkForMessages (node:internal/modules/esm/worker:138:28)
    at process.processImmediate (node:internal/timers:478:21) {
  code: 'ERR_INVALID_URL_SCHEME'
}
Error:
Failed to execute config file: Failed to execute config file
```

Same failure I'm getting in trying to use a `graphql.config.ts` file in my own project.

So something is broken. 

```
$ node --version
v20.14.0
```

Switched to 20.18.1:

```
$ node --version
v20.18.1
```

Same result after `rm -rf node_modules` and `npm i`.

Posting FYI; my next step is to switch to a different GraphQL generator. Sorry.

**Comment by uhyo** [comment]
Thanks for the report. That's a bad news but I'll take a look anyway. 🥲

**Comment by uhyo** [comment]
Note for myself: probably a Windows-only bug

**Comment by uhyo** [comment]
Fixed in v1.7.1. Thanks!
